therapy/etl/manual_override.csv

```diff
@@ -0,0 +1,9 @@
+source,concept_id,field,value
+Wikidata,wikidata:Q412920,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q3995913,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q412616,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q415392,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q415392,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q412323,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q415264,aliases,Ig gamma-1 chain C region
+Wikidata,wikidata:Q418702,aliases,Ig gamma-1 chain C region
```

Comment:
for updating override rules, what should be the standard for updating these going forward? Should we just submit weird findings as additional tickets like the IgG case?